docs/starters.yml

```diff
@@ -1634,3 +1634,20 @@
     - GSAP ready
     - Nice devtools
     - Decorators
+- url: https://gatsby-sseon-starter.netlify.com/
+  repo: https://github.com/SeonHyungJo/gatsby-sseon-starter
+  description: Simple starter template for Gatsby
+  tags:
+    - Simple
```

Comment:
Simple is not a currently in our tag list. Could you please remove it so we don't bloat our list on the website.

```suggestion
```